guide/app_hotkeys.tex

```diff
@@ -352,6 +352,7 @@ \subsection{Solar System Editor}
 \toprule
 \emph{Shortcut key}	& \emph{Description}\\\midrule
 \key{\ctrl+Alt+S}	& Import orbital elements in MPC format... \\
+\key{\ctrl+Alt+Shift+S}	& Open Solar System Editor \\
```

Comment:
I am really not sure we need more than one hotkey action to bring this dialog up. One is fine to avoid the long path through settings/plugins/configure, but how often do you (or the average user) really use it?

Reply:
It's not "_more than one hotkey_" - the main dialog has _no_ hotkey, that's the point.


For me, these should be the actions:

- ctl-alt-S: open SSE
- ctl-alt-shf-S: open import window

But I guess most users are too used to the existing MPC import shortcut, and anyone can use the keyboard shortcut editor if needed.